silhouette/Graphtec.py

```diff
@@ -141,8 +141,9 @@
 SILHOUETTE_CAMEO4_TOOL_EMPTY = 0
 SILHOUETTE_CAMEO4_TOOL_BLADE = 1
```

Comment:
I think 1 is Ratchet blade (black adapter), so it might make sense to rename it correspondingly to `RATCHETBLADE`. Or is there another blade from Cameo3 that is reported as 1?